src/components/Chat/Chat/Chat.ts

```diff
@@ -0,0 +1,20 @@
+import Block from '../../../utils/Block';
+import template from './template.pug'
+
+interface ChatProps {
+    messageText: string;
+    sender: string;
+    date: string;
+    count: string;
+}
+
+
+export default class Chat extends Block {
+    constructor(props: ChatProps) {
+        super(props)
+    }
+
+    render() {
+        return this.compile(template, {  ...this.props  })
```

Comment:
Вообщем стоит пробежаться по коду и поправить, чтобы держать код в чистоте и порядке.
```suggestion
import template from './template.pug';

interface ChatProps {
    messageText: string;
    sender: string;
    date: string;
    count: string;
}


export default class Chat extends Block {
    constructor(props: ChatProps) {
        super(props);
    }

    render() {
        return this.compile(template, {  ...this.props  });
```